quit/application.py

```diff
@@ -248,7 +252,7 @@ def parseArgs(args):
     parser.add_argument('--flask-debug', action='store_true')
     parser.add_argument('--defaultgraph-union', action='store_true')
     parser.add_argument('-f', '--features', nargs='*', action=FeaturesAction,
-                        default=Feature.Unknown,
+                        default=feature_default,
```

Comment:
The features should be set using the command line arguments instead of environment variables, if this is possible.

Reply:
Is it possible to set the environment variables using Docker? I only found the git-way to set this environment variable.

Reply:
@seebi could you please try if you can use the `--feature` command line option instead of the environment variables in the orchestration (https://github.com/AKSW/QuitStore#command-line-options)?

Reply:
yes, i can overwrite it in our compose file